daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/SortedLookupList.scala

```diff
@@ -8,7 +8,8 @@ import ScalazEqual.{equalBy, orderBy}
 import scalaz.{Applicative, Equal, Order, Traverse}
 import scalaz.std.tuple._
 import scalaz.std.string._
-import scalaz.syntax.traverse._
+import scalaz.syntax.functor0._
```

Comment:
What exactly is the difference between `functor0` and `functor`? (and same for traverse)

Reply:
So far I understand from [here](https://github.com/scalaz/scalaz/pull/1479) this seems to be the `functor` only import instead of getting transitive dependencies of a functor too.

Reply:
Exactly.